tasks/yum/start.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: "Start Jenkins"
+  service: name="jenkins" state="started"
+- name: "Wait for Jenkins to come up"
+  uri:
+    url: "http://{{ jenkins_url }}:{{ jenkins_port }}"
```

Comment:
The `jenkins_url` var has "http://" already in it so we shouldn't add it again here, see where the var is defined [here](https://github.com/emmetog/ansible-jenkins/blob/master/defaults/main.yml#L4)